### DISCUSSION TITLE
What version of Apache Subversion the current version of SharpSVN is built against?

### DISCUSSION THREAD

**Discussion by bahrep** [AmpScm/SharpSvn#83] [Q&A] [answered]
Hello,

* What version of Apache Subversion the current version of SharpSVN is built against?

    or

* What version of Apache Subversion the current version of SharpSVN is functionally equivalent to?

Thank you!

**Comment by rhuijben** [comment, marked as answer]
If you see a SharpSvn release of 1.14001.x.y It is build against Subversion 1.14.1. The first digits are Subversion X.Y.Z -> X. <1000*Y + Z>
to allow having a Subversion release and SharpSvn in a standard .Net/Windows release number with 3 significant digits.
